src/storage/manager.rs

```diff
@@ -35,6 +35,7 @@ pub struct StateManager {
 pub struct Status {
     pub available: Ranges,
     pub downloading: Ranges,
+    pub unavailability_map: Vec<u8>,
```

Comment:
It's not clear what this numbers mean. Can it be `Vec<bool>`?

Reply:
done